src/Native/Unix/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ add_compile_options(-Wno-unused-local-typedef)
 add_compile_options(-Wno-c99-extensions)
 add_compile_options(-Wno-c11-extensions)
 add_compile_options(-Wmissing-prototypes)
+add_compile_options(-Wno-incompatible-pointer-types-discards-qualifiers)
```

Comment:
Is there no way to fix the code instead? The warning disabling should be the last resort.

Reply:
In Alpine 3.7, Ubuntu 18 and many other os/distro/versions, that are shipping with clang 5+, this is not required. This is only for clang4 and below that assigning `const` to non-const raises this warning, and -werror renders it as error.